packages/imperative/src/plugins/cmd/install/install.handler.ts

```diff
@@ -96,8 +96,10 @@ export default class InstallHandler implements ICommandHandler {
           "at your own risk. CA Technologies makes no warranties regarding the use of\n" +
           "3rd party plug-ins.\n\n" +
           "Imperative's plugin installation program handles peer dependencies for modules\n" +
-          "in the @brightside namespace, so you can safely ignore NPM warnings about\n" +
-          "missing peer dependencies related to @brightside modules.\n"
+          "in the @brightside namespace and missing package.json file,\n" +
+          "so you can safely ignore NPM warnings about\n" +
+          "missing peer dependencies related to @brightside modules and absent " +
+          PMFConstants.instance.PLUGIN_INSTALL_LOCATION + "package.json file."
```

Comment:
Tested again today and saw this message now:

```
Imperative's plugin installation program handles peer dependencies for modules
in the @brightside namespace and missing package.json file,
so you can safely ignore NPM warnings about
missing peer dependencies related to @brightside modules and absent C:\dev\.zowe\plugins\installedpackage.json file.
```

you should do this: `path.join(PMFConstants.instance.PLUGIN_INSTALL_LOCATION, "package.json")` so that the path is formatted correctly.

I did talk with @BrandonJenkins14 today and he says that the wording is good so no changes there. So make this change and we should be good to merge this. I still have some reservations about the approach, like how often might we have to update our npm dependency to maintain compatibility with the registry, but your comments at least make it easy to get up to speed with how the code works.

On a side note, we should probably be using `TextUtils.wordWrap` instead of manually making the line wraps. I will open a separate issue for this.

Reply:
Thank you for suggestions, last commit should fix that.